File: app/src/main/java/com/formichelli/dfnightselfies/DFNightSelfiesMainActivity.java
package com.formichelli.dfnightselfies;

import android.app.Activity;
import android.os.Bundle;
import android.view.KeyEvent;
import android.view.Window;
import android.view.WindowManager;

import com.google.android.gms.ads.MobileAds;

public class DFNightSelfiesMainActivity extends Activity {
    DFNightSelfiesMainFragment fragment;

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        WindowUtil.setupWindow(this);

        MobileAds.initialize(getApplicationContext(), getString(R.string.banner_ad_unit_id));

        fragment = new DFNightSelfiesMainFragment();

        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, fragment)
                .commit();
    }

    @Override
    public boolean onKeyDown(int keyCode, KeyEvent event) {
        return fragment.onKeyDown(keyCode);
    }

    @Override
    public boolean onKeyUp(int keyCode, KeyEvent event) {
        return fragment.onKeyUp(keyCode);
    }
}

File: app/src/main/java/com/formichelli/dfnightselfies/DFNightSelfiesMainActivity.kt
package com.formichelli.dfnightselfies

import android.app.Activity
import android.os.Bundle
import android.view.KeyEvent
import com.formichelli.dfnightselfies.util.Util

import com.google.android.gms.ads.MobileAds

open class DFNightSelfiesMainActivity : Activity() {
    private val fragment: DFNightSelfiesMainFragment = DFNightSelfiesMainFragment()

    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        Util.setupWindow(this)

        MobileAds.initialize(applicationContext, getString(R.string.banner_ad_unit_id))

        fragmentManager
                .beginTransaction()
                .replace(android.R.id.content, fragment)
                .commit()
    }

    override fun onKeyDown(keyCode: Int, event: KeyEvent) = fragment.onKeyDown(keyCode)

    override fun onKeyUp(keyCode: Int, event: KeyEvent) = fragment.onKeyUp(keyCode)

    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<String>, grantResults: IntArray) = fragment.onRequestPermissionsResult(requestCode, permissions, grantResults)
}
